install/cupy_builder/_features.py

```diff
@@ -462,16 +468,17 @@ def __init__(self, ctx: Context):
             'curand.h',
             'cusparse.h',
         ]
-        # TODO(kmaehashi): Split profiler module so that dependency to
-        # `cudart` can be removed when using CUDA Python.
         self.libraries = (
-            (['cudart'] if ctx.use_cuda_python else ['cuda', 'cudart']) + [
-                'cublas',
-                'cufft',
-                'curand',
-                'cusparse',
-            ]
+            # CUDA Driver
+            ([] if ctx.use_cuda_python else ['cuda']) +
+
+            # CUDA Runtime
+            _cudart_static_libs +
+
+            # CUDA Toolkit
+            ['cublas', 'cufft', 'curand', 'cusparse']
         )
+        self.static_libraries = ['cudart_static']
```

Comment:
Q: Would this make the CUDA Python build also link to cudart statically?

Reply:
I believe ld is be clever enough to avoid linking static libs if symbols are unused 😄

Reply:
Yes I was asking a silly question. CUDA Python does not use any cudart symbols, and indeed the linker would do nothing.